opsimulator/derive.go

```diff
@@ -5,7 +5,7 @@ import (
 	"github.com/ethereum/go-ethereum/core/types"
 )
 
-func executingMessagePayloadBytes(log *types.Log) []byte {
```

Comment:
Lets move this function into `interop` so that the package can be standalone

Reply:
since both of the functions in this package are specific to interop, I moved `derive.go` into the `interop` package